src/layouts/styles.scss

```diff
@@ -0,0 +1,10 @@
+@import "~bulma";
+
+.gallery-container .albums {
+  flex-wrap: wrap;
+  justify-content: space-around;
```

Comment:
If you just use Bulma columns without this, all the albums should have fixed width and height even on resizing. It's probably because of this